src/main/kotlin/uk/gov/gdx/datashare/repository/EventPublisher.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.gdx.datashare.repository
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDateTime
+
+data class EventPublisher(
+  @Id
+  val id: Long,
+  val publisherName: String,
+  val whenCreated: LocalDateTime? = null,
```

Comment:
should these whenCreated fields be nullable? I feel like they should always have a value